src/containers/layouts/auth/signUp3/signUp3.component.tsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import {
+  ButtonProps,
+  ImageBackground,
+  ImageProps,
+  ImageSourcePropType,
+  View,
+} from 'react-native';
+import {
+  StyleType,
+  ThemedComponentProps,
+  ThemeType,
+  withStyles,
+} from '@kitten/theme';
+import {
+  SignUpForm2,
+  SignUpForm2Type,
+} from '@src/components/auth';
+import { ProfilePhoto } from '@src/components/social';
+import {
+  AvoidKeyboard,
+  Button,
+  Text,
+} from '@src/components/common';
+import { PlusIcon } from '@src/assets/icons';
+
+interface ComponentProps {
+  onSignUpPress: (formValue: SignUpForm2Type) => void;
+  onSignInPress: () => void;
+  onPhotoPress: () => void;
+}
+
+export type SignUp3Props = ThemedComponentProps & ComponentProps;
+
+class SignUp3Component extends React.Component<SignUp3Props> {
+
+  private backgroundImage: ImageSourcePropType = {
+    uri: `https://images.unsplash.com/photo-1536431311719-398b6704d4cc?ixlib=rb-1.2
+    .1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=934&q=80`,
+  };
+
+  private onPhotoButtonPress = () => {
+    this.props.onPhotoPress();
+  };
+
+  private onSignInButtonPress = () => {
+    this.props.onSignInPress();
+  };
+
+  private onSignUpButtonPress = (formValue: SignUpForm2Type) => {
+    this.props.onSignUpPress(formValue);
+  };
+
+  private renderPhotoButtonIcon = (style: StyleType): React.ReactElement<ImageProps> => {
+    const { themedStyle } = this.props;
+
+    return PlusIcon({ ...style, ...themedStyle.photoButtonIcon });
+  };
+
+  private renderPhotoButton = (): React.ReactElement<ButtonProps> => {
+    const { themedStyle } = this.props;
+
+    return (
+      <Button
+        style={themedStyle.photoButton}
+        icon={this.renderPhotoButtonIcon}
+        onPress={this.onPhotoButtonPress}
+      />
+    );
+  };
+
+  public render(): React.ReactNode {
+    const { themedStyle } = this.props;
+
+    return (
+      <AvoidKeyboard>
+        <ImageBackground
+          style={themedStyle.container}
+          source={this.backgroundImage}>
+          <View style={themedStyle.headerContainer}>
+            <ProfilePhoto
+              style={themedStyle.photo}
+              resizeMode='center'
+              source={{ uri: 'https://akveo.github.io/eva-icons/fill/png/128/person.png' }}
```

Comment:
Does it make sense to move it to the "assets" folder and use like 'calling' method with no styles?